content/clusters/21K.Omicron.md

```diff
@@ -23,6 +35,8 @@ There are also two mutations in Nucleocapsid, <AaMut mut={'N:R203K'}/> and <AaMu
 
 Please note that non-synonymous mutations (in the right-hand menu) have not yet been calculated - this will be updated as soon as sequences are finished processing.
 
+- Prior to the variant's official designation as <Who name="Omicron" />, some reports prematurely referred to the variant as 'Nu,' which would have have been the next letter after <Who name="Delta" />. WHO apparently skipped 'Nu' because of confusion with the English word 'new' and 'Xi' because it is a common surname ([Kai Kupferschmidt, Twitter](https://twitter.com/kakape/status/1464671345070186497))
```

Comment:
Should this be "Mu"?

```suggestion
- Prior to the variant's official designation as <Who name="Omicron" />, some reports prematurely referred to the variant as 'Nu,' which would have have been the next letter after <Who name="Mu" />. WHO apparently skipped 'Nu' because of confusion with the English word 'new' and 'Xi' because it is a common surname ([Kai Kupferschmidt, Twitter](https://twitter.com/kakape/status/1464671345070186497))
```

Reply:
You're right!! Thank you for catching!